### PR TITLE
Lead space mvp for patterns section

### PR DESCRIPTION
**Pull request by JennySanchez**
### Related Ticket(s)

{{Provide url(s) to the related ticket(s) that this pull request addresses}}

### Description

{{Add a human-readable description / detail summary of what the PR is changing and any details around how and why}}

{{If applicable, include a screenshot indicating an example or examples of what the PR is changing in the application}}

### Changelog

**New**

- {{new thing}}

**Changed**

- {{changed thing}}

**Removed**

- {{removed thing}}

### CODE REVIEW

File: src/pages/patterns/lead-space.mdx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+Is the page you are creating used for presenting typographical content for users to gain insight or for helping users to focus on a task?
+
+#### Centered VS left-aligned
+No matter which segments your offers are in the user journey, always make sure you have a clear understanding of the purpose of the page. See the Content alignment section below for detail.
+
+We are always refining and testing the standards, so please contribute by working with us to refine the standard details.
+
+## Size standards
+
+The curated Lead Space size standards are super, tall, regular, short, mini. please start by using the sizes listed here so we can create consistency for the IBM.com experience.
```
Comment (wonilsuhibm) [inline src/pages/patterns/lead-space.mdx:51]: Let's add some messages around using the right size for the right purpose. My suggestion:

"The curated Lead Space size standards are super, tall, regular, short, mini. Choose the right size for the purpose of the lead space on the given page."

Comment (JennySanchez) [reply src/pages/patterns/lead-space.mdx]: Sounds good. FYI: We use Lead space in the sentence when calling out a component or pattern. Let me know if there are any changes to that rule I am not aware of.

**Comment by wonilsuhibm** [comment]
Updates have been made, here are the previews:

[Lead space pattern page](https://ibmdotcom-library.s3-web.us-south.cloud-object-storage.appdomain.cloud/deploy-previews/729/patterns/lead-space/index.html)

[Lead space component page](https://ibmdotcom-library.s3-web.us-south.cloud-object-storage.appdomain.cloud/deploy-previews/729/components/leadspace/index.html)

**Comment by JennySanchez** [comment]
@wonilsuhibm Both Lead Space page content looks good to me.

**Comment by oliviaflory** [comment]
## Lead space pattern page

### Anatomy
It would be nice to update the anatomy example to use the numbers with a list associated at the bottom. 
_Not a blocker for initial release._

### Height standards
Not all the lead spaces are available in varying heights, so might be a little confusing. Should we link to the storybook examples in the chart of sizes? 


### Imagery
**Image sizing**
Should we recommend that they always upload a size that accommodates the Retina display instead of showing the standard display sizing? 

### Call to action
**Single CTA**
Should we remove "Learn more" as an example? We say in the previous section that Learn more is generic and not informative.


## Lead space component page
### Intro
Suggestion: remove generic "here" link and use "Lead space pattern" as the link to the patterns documentation. 
Also pattern should not be plural (I don't think)

> For detailed guidelines around the general lead space pattern, please see the **Lead space pattern** page.


### Color themes
Technically... all the themes are available, right? 

### Left aligned style options
**Content guidance**
1. Are the images sizes for standard or retina? 
2. Should the desktop recommendation match the size on the pattern page, they do not currently.

**Type styles**
IMO listing out which type style is used is slightly confusing because it's not something the users choose in the knobs of the lead space, but rather the variant. Maybe adding the variant name would help?

`Leads pace with display-01` image is missing/not linked


### Overall comments
1. Missing design and functional specifications section, I feel that is necessary for adopters to link to our Box folder
2. We are missing the feedback section.
3. Should development documentation be moved below Design and functional specifications to maintain the same order of documentation for every component, following the template.

**Comment by wonilsuhibm** [comment]
@oliviaflory @JennySanchez updates made. please take a look.

**Comment by JennySanchez** [comment]
@wonilsuhibm Found two items to share with you for a second look.

1. Left-aligned style options/Type styles/Second example
Since it says "most effective when there is no call to action", should we take the call to action buttons off the image example?

2. Specificationsm - "$expressive-heading-06 at 1312 xlg breakpoint" and "$display-01 at 1312 xlg breakpoint"
Both bullet point 6/Background image: need to take the "." at the end of the sentence off

File: src/pages/patterns/lead-space.mdx [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+<Column colMd={4} colLg={4} >
+
+![Lead space content alignment visual](./images/lead-space/lead-space-content-alignment-example-1-3.png)
+
+</Column>
+</Row>
+
+## Height standards
+
+The size of the lead space usuall reflects the intent of the page and sets the right expectation for the user. Choose from these heights for the different purposes and use cases. TO check out the available lead space heights, see the lead space component pages linked [below](#related-components-and-patterns).
```
Comment (oliviaflory) [inline src/pages/patterns/lead-space.mdx:86]:
```suggestion
The size of the lead space usuall reflects the intent of the page and sets the right expectation for the user. Choose from these heights for the different purposes and use cases. To check out the available lead space heights, see the lead space component pages linked [below](#related-components-and-patterns).
```

File: src/pages/patterns/lead-space.mdx [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+title: Lead space
+description: A Lead space is positioned at the top of a web page and serves as the first site of viewer engagement. Its primary purpose is to orient the user when they land on a page, inform them of the content, and guide them to the first key piece of content on the page, whether it’s a crucial piece of information or a call-to-action.
+  
+---
+
+<PageDescription>
+
+A Lead space is positioned at the top of a web page and serves as the first site of user engagement. Its primary purpose is to orient the user when they land on a page, inform them of the content, and guide them to the first key piece of content on the page, whether it's a crucial piece of information or a call-to-action.
+
+When designed and implemented effectively, the lead space can be used to engage visitors immediately in one of the most important moments in the user’s journey through a complex web environment. It is the first section below the masthead and can include a stylized background, headline, supporting message and call to action to accommodate user needs.
```
Comment (oliviaflory) [inline src/pages/patterns/lead-space.mdx:11]:
```suggestion
When designed and implemented effectively, the Lead space can be used to engage visitors immediately in one of the most important moments in the user’s journey through a complex web environment. It is the first section below the masthead and can include a stylized background, headline, supporting message and call to action to accommodate user needs.
```

File: src/pages/patterns/lead-space.mdx [hunk 4]
```diff
@@ -0,0 +1,240 @@
...
+<AnchorLink>Imagery</AnchorLink>
+<AnchorLink>Call to action</AnchorLink>
+<AnchorLink>Pattern in use</AnchorLink>
+<AnchorLink>Related components and patterns</AnchorLink>
+
+</AnchorLinks>
+
+## Anatomy 
+
+Here are some of the elements that are commonly used and currently available. As we continue to build and offer additioanl options, we will share new designs on this page.
```
Comment (oliviaflory) [inline src/pages/patterns/lead-space.mdx:41]:
```suggestion
Here are some of the elements that are commonly used and currently available. As we continue to build and offer additional options, we will share new designs on this page.
```

File: src/pages/patterns/lead-space.mdx [hunk 5]
```diff
@@ -0,0 +1,240 @@
...
+
+![Anatomy of the lead space](./images/lead-space/LeadSpace_Left_Heading06_Anatomy.jpg)
+
+## Layout
+
+### Best practices
+
+Here are some best practices when it comes to layout in lead space:
```
Comment (oliviaflory) [inline src/pages/patterns/lead-space.mdx:49]:
```suggestion
Here are some best practices when it comes to layout in Lead space:
```

File: src/pages/patterns/lead-space.mdx [hunk 6]
```diff
@@ -0,0 +1,240 @@
...
+
+For actual implementations, please see the component pages linked [below](#related-components-and-patterns).
+
+### Content alignment
+
+On IBM.com pages, there are two layout options&mdash;left-aligned and center-aligned. Follow these guidelines to select the right layout for your page.
+
+1. Align to the primary content of the given page.
+
+2. When there is a background image in the lead space, using left-aligned layout is recommended regardless of the primary content alignement.
```
Comment (oliviaflory) [inline src/pages/patterns/lead-space.mdx:64]:
```suggestion
2. When there is a background image in the Lead space, using left-aligned layout is recommended regardless of the primary content alignment.
```

File: src/pages/patterns/lead-space.mdx (hunk 2 above)
Comment (oliviaflory) [inline src/pages/patterns/lead-space.mdx]:
```suggestion
The size of the Lead space usually reflects the intent of the page and sets the right expectation for the user. Choose from these heights for the different purposes and use cases. To check out the available Lead space heights, see the Lead space component pages linked [below](#related-components-and-patterns).
```

File: src/pages/patterns/lead-space.mdx [hunk 7]
```diff
@@ -0,0 +1,240 @@
...
+## Height standards
+
+The size of the lead space usuall reflects the intent of the page and sets the right expectation for the user. Choose from these heights for the different purposes and use cases. TO check out the available lead space heights, see the lead space component pages linked [below](#related-components-and-patterns).
+
+| Height | Recommended usage |
+| :--- | :--- |
+| 560px and taller | Brand or immersive experience centric pages |
+| 480px, 400px | Branding and messaging centric pages |
+| 320px | Functional or navigational pages |
+| 240px | Minimal lead space with simple title |
```
Comment (oliviaflory) [inline src/pages/patterns/lead-space.mdx:93]:
```suggestion
| 240px | Minimal Lead space with simple title |
```

**Comment by wonilsuhibm** [comment]
@JennySanchez Thank you.

1. I am not sure if we have data to back that claim that it's more effective when there is no CTA. I'm removing it.

2. Do we need a period at the end of a phrase?

File: src/pages/patterns/lead-space.mdx [hunk 8]
```diff
@@ -0,0 +1,240 @@
...
+
+
+## Content
+
+* Copy in the lead space creates an immediate impact while informing web visitors about the page or service quickly and effectively.
```
Comment (oliviaflory) [inline src/pages/patterns/lead-space.mdx:98]:
```suggestion
* Copy in the Lead space creates an immediate impact while informing web visitors about the page or service quickly and effectively. 
```

File: src/pages/patterns/lead-space.mdx [hunk 9]
```diff
@@ -0,0 +1,240 @@
...
+| 480px, 400px | Branding and messaging centric pages |
+| 320px | Functional or navigational pages |
+| 240px | Minimal lead space with simple title |
+
+
+## Content
+
+* Copy in the lead space creates an immediate impact while informing web visitors about the page or service quickly and effectively. 
+* Copy should be brief and relevant. This will help to orient and inform the user as quickly as possible.
+* Don’t rely on the lead space to deliver the majority of the messaging and information about your topic or offering. The rest of the page is for delivering your message.
```
Comment (oliviaflory) [inline src/pages/patterns/lead-space.mdx:100]:
```suggestion
* Don’t rely on the Lead space to deliver the majority of the messaging and information about your topic or offering. The rest of the page is for delivering your message.
```

File: src/pages/patterns/lead-space.mdx [hunk 10]
```diff
@@ -0,0 +1,240 @@
...
+* A lead space is not required for all scenarios; follow the guidance for the page type you are creating.
```
Comment (oliviaflory) [inline src/pages/patterns/lead-space.mdx:101]:
```suggestion
* A Lead space is not required for all scenarios; follow the guidance for the page type you are creating.
```

File: src/pages/patterns/lead-space.mdx [hunk 11]
```diff
@@ -0,0 +1,240 @@
...
+* Portrait photography
+* Landscape photography
+* Illustration
+
+Usage: Images should never impact the readability or legibility of the Lead space copy. 
+
+* Where possible, images should blend into the background to avoid hard lines or optical divisions across the Lead space. 
+* Consider using a gradient overlay to darken the left area of a background image, if it impacts the text. 
+* If using a full-width image in the background, the image should be larger in size than the Lead space size, to allow for browser resizing. If you choose this option, include additional padding around the image.
+* Only use one image per lead space.
```
Comment (oliviaflory) [inline src/pages/patterns/lead-space.mdx:116]:
```suggestion
* Only use one image per Lead space.
```

File: src/pages/patterns/lead-space.mdx [hunk 12]
```diff
@@ -0,0 +1,240 @@
...
+</Column>
+</Row>
+
+#### Image sizing
+
+* For a standard screen, the recommended width for a full-width background image is 1400 pixel minimum, and the size should not exceed 200KB. 
+* For a retina display, the recommended width for a full-width background image is 2800 pixels and the size should not exceed 200KB.
+
+## Call to action
+The maximum number of CTAs per Lead space is two. Note, however, that it is not required to use a call to action in a lead space.
```
Comment (oliviaflory) [inline src/pages/patterns/lead-space.mdx:142]:
```suggestion
The maximum number of CTAs per Lead space is two. Note, however, that it is not required to use a call to action in a Lead space.
```

File: src/pages/patterns/lead-space.mdx [hunk 13]
```diff
@@ -0,0 +1,240 @@
...
+
+* Sign up
+* Log in
+* Register today
+* Download the smartpaper
+
+Usage: Use a primary button only. Icon is optional.
+
+Caution: Using a CTA in your Lead space will direct your users to a new section, modal or page. 
+If you do not want users to leave the page before discovering information below, consider using a lead space without any CTAs. Keep in mind that scroll depth is a valuable KPI; driving users off your page as soon as they land can be counterproductive.
```
Comment (oliviaflory) [inline src/pages/patterns/lead-space.mdx:179]:
```suggestion
If you do not want users to leave the page before discovering information below, consider using a Lead space without any CTAs. Keep in mind that scroll depth is a valuable KPI; driving users off of your page as soon as they land can be counterproductive.
```

**Review by oliviaflory — COMMENTED** [review]
added comments inline for all "lead" changed to "Lead" per guidelines

File: src/pages/patterns/lead-space.mdx (hunk 2 above)
Comment (wonilsuhibm) [reply src/pages/patterns/lead-space.mdx]: Addressed separately in this commit: https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/pull/729/commits/65d22c3886ec45d20c984d311f848b002cfc4d4d

**Comment by JennySanchez** [comment]
@wonilsuhibm 

Do we need a period at the end of a phrase?
In this use case, spec description, **we do not**. I am suggesting to take the period **off** in both places.

**Comment by wonilsuhibm** [comment]
@JennySanchez the examples in your comment look like complete sentences—we most likely need to keep that period there.

**Comment by JennySanchez** [comment]
@wonilsuhibm ok.

File: src/pages/patterns/lead-space.mdx [hunk 14]
```diff
@@ -0,0 +1,240 @@
...
+<Column colMd={4} colLg={4} >
+
+![Lead space content alignment visual](./images/lead-space/lead-space-content-alignment-example-1-3.png)
+
+</Column>
+</Row>
+
+## Height standards
+
+The size of a lead space usually reflects the intent of the page and sets the right expectation for the user. Choose from these heights for the different purposes and use cases. To check out the available Lead space heights, see the Lead space component pages linked [below](#related-components-and-patterns).
```
Comment (oliviaflory) [inline src/pages/patterns/lead-space.mdx:86]:
```suggestion
The size of a Lead space usually reflects the intent of the page and sets the right expectation for the user. Choose from these heights for the different purposes and use cases. To check out the available Lead space heights, see the Lead space component pages linked [below](#related-components-and-patterns).
```

**Review by oliviaflory — APPROVED** [review]

**Pull request merged**